docs/Telemetry.md

```diff
@@ -185,34 +185,36 @@ Refer to the below table to understand different critical telemetry events raise
 | `ProcessingHTMLTextMiddlewareFailed`     |On HTML Field adding noopener noreferrer failed |
 | `ProcessingSanitizationMiddlewareFailed`     |On HTML Field sanitization failed |
 | `FormatTagsMiddlewareJSONStringifyFailed`     |On ACS Format Message Tags Middleware failed |
-|`IC3ThreadUpdateEventReceived`|On `IC3 ThreadUpdateEvent` Received|
-|`AverageWaitTimeMessageRecieved`|On Average Wait Time system message Received|
-|`QueuePositionMessageRecieved`|On Queue Position system message Received|
-|`IC3ThreadUpdateEventReceived`|On `IC3 ThreadUpdateEvent` Received|
```

Comment:
Please remove this duplicate event (Line 188 and 191)

Reply:
will address this on separate PR